lib/hanami/reloader/commands.rb

```diff
@@ -1,9 +1,6 @@
 # frozen_string_literal: true
 
 require "hanami/cli"
-require "hanami/cli/command"
-require "hanami/cli/commands"
-require "hanami/cli/commands/app/server"
```

Comment:
Used the zeitwerkify branch of hanami-cli here, just to demonstrate this nice outcome :)